input.lisp

```diff
@@ -262,20 +263,22 @@ match with an element of the completions."
          (full-string-width (+ string-width space-width))
          (pos (input-line-position input))
          (width (+ prompt-width
-                   (max 100 (+ full-string-width space-width tail-width)))))
+                   (max 100 (+ full-string-width space-width tail-width))))
+         (text-y (+ (font-ascent (screen-font screen))
```

Comment:
Maybe `height` would be a better name?

Reply:
I'm not sure if `height` is better, since the variable shows the position of the text, not the height of the window. The window height is set earlier in `SETUP-INPUT-WINDOW` to be fontheight+padding*2 (it doesn't need to be changed when redrawing since the input window is always just one line high).

Reply:
Thanks for clearing that up

Reply:
Looks good to me!